syntax = "proto3";

service LoggingService {
  rpc LogMessage (LogRequest) returns (LogResponse);
  rpc GetLogs (Empty) returns (LogResponse);
}

message LogRequest {
  string id = 1;
  string msg = 2;
}

message LogResponse {
  string status = 1;
}

message Empty {}
